### PR TITLE
fixed serial

### PR DESCRIPTION
**Pull request by dmmat**

### CODE REVIEW

File: uno/uno.ino [hunk 1]
```diff
@@ -36,7 +38,10 @@ void setup() {
 
 void loop() {
   if (threadSensors.shouldRun()) { threadSensors.run(); }
-  while (Serial.available()) { updateState(Serial.readString()); }
+  while (Serial.available()) {
+    command = Serial.readStringUntil('\n');
```
Comment (elhowm) [inline uno/uno.ino:45]: where is `updateState` by the command we read?

**Review by elhowm — CHANGES_REQUESTED** [review]

**Comment by dmmat** [comment]
oh shit

File: uno/uno.ino [hunk 2]
```diff
@@ -36,7 +38,14 @@ void setup() {
 
 void loop() {
   if (threadSensors.shouldRun()) { threadSensors.run(); }
-  while (Serial.available()) { updateState(Serial.readString()); }
+
+  while (Serial.available()) {
+    command = Serial.readStringUntil('\n');
+  };
+
+  if (command != "") { updateState(command); };
```
Comment (elhowm) [inline uno/uno.ino:48]: need `command = ""` after update because you will loop `updateState` on first time command received

**Review by elhowm — CHANGES_REQUESTED** [review]

File: uno/uno.ino (hunk 2 above)
Comment (dmmat) [reply uno/uno.ino:48]: Ok, added

**Review by elhowm — APPROVED** [review]

**Pull request merged**